Tests/MongoSwiftTests/MongoDatabaseTests.swift

```diff
@@ -112,26 +113,74 @@ final class MongoDatabaseTests: MongoSwiftTestCase {
         )
 
         expect(try db.createCollection("fooView", options: viewOptions)).toNot(throwError())
-        let decoder = BSONDecoder()
-        var collectionInfo = try db.listCollections().map { try decoder.decode(CollectionInfo.self, from: $0) }
+
+        var collectionInfo = try Array(db.listCollections()) as [CollectionSpecification]
```

Comment:
nit: do we need the type annotation? I'd think it would be inferred by the return type of `listCollections`